### PR TITLE
CORE-10120 Update e2e-test utils for custom metadata tests

### PR DESCRIPTION
**Pull request by YashNabar**
Updates E2E test utilities to support custom metadata testing in static networks.
https://github.com/corda/corda-e2e-tests/pull/175

### CODE REVIEW

File: testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt [hunk 1]
```diff
@@ -100,15 +100,17 @@ class ClusterBuilder {
     fun cpbUpload(resourceName: String) = uploadUnmodifiedResource("/api/$REST_API_VERSION_PATH/cpi/", resourceName)
 
     /** Assumes the resource is a CPB and converts it to CPI by adding a group policy file */
```
Comment (nikinagy) [inline testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt]: Can we update the kdocs?

File: testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt [hunk 2]
```diff
@@ -258,10 +271,13 @@ class ClusterBuilder {
      * notary service. This is fine for now as we only support a 1-1 mapping from notary service to
      * notary vnode. It will need revisiting when 1-* is supported.
```
Comment (nikinagy) [inline testing/e2e-test-utilities/src/main/kotlin/net/corda/e2etest/utilities/ClusterBuilder.kt:286]: Kdocs update here too

**Review by nikinagy — COMMENTED** [review]
Looks good to me. Just left some NIT about updating the kdocs to reflect the changes being made.

**Review by nikinagy — APPROVED** [review]
LGTM

**Review by charlieR3 — APPROVED** [review]

**Pull request merged**